### PR TITLE
Add Person entity to search API

### PR DESCRIPTION
**Pull request by acsehi**

### CODE REVIEW

**Comment by cristianv-ms** [comment]
Please solve the warnings in this files:
https://github.com/microsoftgraph/microsoft-graph-docs/blob/ancsehi-ms-GraphDocUpdate/api-reference/beta/resources/printerstatus.md

https://github.com/microsoftgraph/microsoft-graph-docs/blob/ancsehi-ms-GraphDocUpdate/concepts/search-concept-person.md

**Comment by nmoreau** [comment]
@acsehi , I have added cristian who should be the reviewer for anything on Public API.
You can add me as optional only.

**Review by cristianv-ms — CHANGES_REQUESTED** [review]
Add to entityType property, the new entity type "person" in the description column, in searchrequest.md file.

**Review by cristianv-ms — APPROVED** [review]

File: concepts/search-concept-person.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: "Use the Microsoft Search API in Microsoft Graph to search people (preview)"
```
Comment (Lauragra) [inline concepts/search-concept-person.md:2]: Please update the toc.yml file in the /concepts folder to surface this new concept topic in the TOC.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**